test/functional/page_objects/common_page.js

```diff
@@ -324,8 +324,9 @@ export function CommonPageProvider({ getService, getPageObjects }) {
     }
 
     async isChromeVisible() {
-      const globalNavShown = await testSubjects.exists('globalNav');
-      const topNavShown = await testSubjects.exists('top-nav');
+      const globalNavShown = (await testSubjects.getAttribute('globalNav', 'aria-hidden')).includes('false');
+      const topNavShown = (await testSubjects.getAttribute('top-nav', 'aria-hidden')).includes('false');
+      log.debug(`isChromeVisible globalNav=${globalNavShown}, top-nav=${topNavShown}`);
       return globalNavShown && topNavShown;
     }
```

Comment:
In my testing, these data-test-subjs are ALWAYS there.  Its the aria-hidden that changes.  I don't know how this could be passing in master.